disco/models/upgrade_cost_analysis_equipment_model.py

```diff
@@ -31,6 +32,12 @@ def from_file(cls, filename: Path):
 
         """
         return cls(**load_data(filename))
+    
+    @validator('*')
```

Comment:
@daniel-thom I'm having a hard time deciding the best way to convert all nan to None.
This validator would need making required float fields optional.
(which could be done for those fields where there is a possibility of nan)